gh-pages/src/_includes/default.html

```diff
@@ -40,7 +40,7 @@
       </a>
 
       <p>
-        Baseline gives you clear information about which web platform features work across browsers.
```

Comment:
Throughout, this PR replaces "clear" with "summarized." I don't love that.

IMO, the main problem we're referring to in #2783 isn't that people don't realize that Baseline is a summary; it's a false-negative problem. If something _is_ Baseline WA, you _can_ use it, but people might wrongly assume that if something is NA or LA, then you simply _can't_ use it, and you'll just have to wait.

If we all agreed that Baseline wasn't "clear," then I think Baseline wouldn't be useful at all, but I think it _is_ useful because it is clear.